README.md

```diff
@@ -40,24 +48,37 @@ make build PACKER_ARGS="-only vagrant"
 make vagrant
 ```
 
-### Usage (docker)
+Docker invocation:
 
-#### Note: vagrant builds are unsupported in Docker, see FAQ
+*Note*: vagrant builds are unsupported in Docker, see FAQ
 
+Set up:
 ```bash
-
 # builds and tags a docker container to run monopacker in
 # monopacker:build by default, can override DOCKER_IMAGE make var
 make dockervalidate
+```
 
+Build images:
+```bash
 # look ma, no dependencies!
-make dockerbuild PACKER_ARGS='-only docker_worker_aws' SECRETS_FILE=./real_secrets.yaml
+make dockerbuild SECRETS_FILE=./real_secrets.yaml
 ```
 
-### Templating packer.yaml for your builders
+You can override a few Make variables (with assignemnts after `make build` or `make dockerbuild`, as in the examples above:
 
-```bash
+* `SECRETS_FILE`: path to a local file containing secrets (see below)
+* `BUILDERS`: builders to set up
+* `PACKER_VARS`: packer variables
+* `PACKER_ARGS`: additional arguments to packer
+
+Note that there are two ways to build only a single builder: `make .. BUILDERS=mybuilder` or `make .. PACKER_ARGS='-only mybuilder'`.
```

Comment:
The example above was intended to demonstrate that you can pass arbitrary arguments to `packer` with `PACKER_ARGS` - so while this is true, the example is not limited to choosing builders.

Reply:
Got it.  I think the rewrite is clear about that, if not by example?  I'm happy to clarify if you'd like.